### PR TITLE
refactor: remove Promise.await

### PR DESCRIPTION
**Pull request by ggazzo**

### CODE REVIEW

File: apps/meteor/app/authorization/server/functions/canSendMessage.ts [hunk 1]
```diff
@@ -59,13 +59,16 @@ async function canSendMessageAsync(
 	return room;
 }
 
+/* deprecated */
```
Comment (sampaiodiego) [inline apps/meteor/app/authorization/server/functions/canSendMessage.ts:62]: do you know this comment means nothing right?

**Review by sampaiodiego — DISMISSED** [review]

File: apps/meteor/ee/app/livechat-enterprise/server/lib/VisitorInactivityMonitor.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ export class VisitorInactivityMonitor {
 			name: this._name,
 			schedule: (parser) => parser.cron(everyMinute),
 			job: () => {
-				Promise.await(this.handleAbandonedRooms());
+				return this.handleAbandonedRooms();
```
Comment (sampaiodiego) [inline apps/meteor/ee/app/livechat-enterprise/server/lib/VisitorInactivityMonitor.ts:46]: have you checked if `SyncedCron` supports `Promises`? I mean, it tracks how much time a cronjob has run, so it kinda waits for the function to execute.. 

what if we let this `Promise.await` until we take a look at the `SyncedCron` package?

Comment (ggazzo) [inline apps/meteor/ee/app/livechat-enterprise/server/lib/VisitorInactivityMonitor.ts:46]:
```suggestion
				return Promise.await(this.handleAbandonedRooms());
```

Comment (ggazzo) [reply apps/meteor/ee/app/livechat-enterprise/server/lib/VisitorInactivityMonitor.ts]: you are right I thought this one was from agenda, the meteor package doenst handle async functions

**Review by sampaiodiego — APPROVED** [review]

**Pull request merged**